src/v/cluster/rm_stm.cc

```diff
@@ -1464,26 +1465,28 @@ void rm_stm::abort_old_txes() {
     });
 }
 
+fragmented_vector<model::producer_identity>
```

Comment:
nit: I guess it is better to use `chunked_vector` here, as it will save memory if the vector is small

Reply:
Good idea, I forgot about chunked_vector, wonder if we should just alias fv to cv